spec/lib/word_counter_spec.rb

```diff
@@ -85,4 +85,9 @@
     word_counter.text = "\“嘿Bob,\” Alice说，‘啊？！？’"
     expect(word_counter.count).to eq(5)
   end
+
+  it "doesn't count parentheses in rp" do
```

Comment:
You can leave out this test, which is not too related to the current issue (the word counter ignores all HTML tags).

Reply:
It's true that this is not a test that justifies the other changes in the PR. It would have passed before. So I agree with leaving it out if that's what you mean.

There doesn't seem to be tests that involve parenthesized text, full-width or not, and parentheses might be common things that would warrant tests, but it seems like that's a separate issue to think about.

(That's a long way to say "yes" but I wanted to make sure I have it right.)

Reply:
Yeah, let's leave this alone for now as it's not related to the main changes in the PR.